src/apps/explorer/pages/Metadata/config.ts

```diff
@@ -0,0 +1,214 @@
+import { AjvError, FormValidation, UiSchema } from '@rjsf/core'
+import {
+  LATEST_APP_DATA_VERSION,
+  LATEST_QUOTE_METADATA_VERSION,
+  LATEST_REFERRER_METADATA_VERSION,
+} from '@cowprotocol/app-data'
+import { JSONSchema7 } from 'json-schema'
+
+const ERROR_MESSAGES = {
+  REQUIRED: 'Required field.',
+  INVALID_ADDRESS: 'This is not an address.',
+  ONLY_DIGITS: 'Only digits are allowed.',
+}
+
+export const INITIAL_FORM_VALUES = {
+  version: LATEST_APP_DATA_VERSION,
+  metadata: {
+    referrer: {},
+    quote: {},
+  },
+}
+
+export const getSchema = async (): Promise<JSONSchema7> => {
+  const { default: latestSchema } = await import(`@cowprotocol/app-data/schemas/v${LATEST_APP_DATA_VERSION}.json`)
```

Comment:
There's an util for this, exposed on app-data and sdk 
https://github.com/cowprotocol/app-data/blob/main/src/utils.ts#L42=
https://github.com/cowprotocol/cow-sdk/blob/main/src/api/metadata/index.ts#L53=